sentences/de/_common.yaml

```diff
@@ -9,12 +9,144 @@ responses:
     # Errors for when user is not logged in
     no_area: "Kein Bereich mit dem Namen {{ area }} vorhanden"
     no_floor: "Keine Etage mit dem Namen {{ floor }} vorhanden"
-    no_domain: "Kein {{ domain }} vorhanden"
-    no_domain_in_area: "Im Bereich {{ area }} ist kein {{ domain }} vorhanden"
-    no_domain_in_floor: "In der Etage {{ floor }} ist kein {{ domain }} vorhanden"
-    no_device_class: "Kein {{ device_class }} vorhanden"
-    no_device_class_in_area: "Im Bereich {{ area }} ist kein {{ device_class }} vorhanden"
-    no_device_class_in_floor: "In der Etage {{ floor }} ist kein {{ device_class }} vorhanden"
+    no_domain: |
+      {% set translations = {
+        "button": "keine Taste",
+        "camera": "keine Kamera",
+        "input_button": "keine Tasteneingabe",
+        "alarm_control_panel": "keine Alarmzentrale",
+        "automation": "keine Automatisierung",
+        "fan": "kein Ventilator",
+        "climate": "kein Thermostat",
+        "humidifier": "kein Luftbefeuchter",
+        "input_boolean": "keine Boolesche Eingabe",
+        "siren": "keine Sirene",
+        "water_heater": "kein Warmwasserbereiter",
+        "light": "keine Leuchte",
+        "switch": "kein Schalter",
+        "script": "kein Skript",
+        "remote": "keine Fernbedienung",
+        "lock": "kein Schloss",
+        "vacuum": "kein Staubsauger",
+        "scene": "keine Szene",
+        "media_player": "kein Media-Player",
+        "lawn_mower": "kein Rasenmäher",
+        "valve": "kein Ventil"
+        } %}
+      {% if domain in translations -%}
+        Es wurde {{ translations[domain] }} gefunden
```

Comment:
I don't like the term `gefunden` because it is not there. Would more like `nicht vorhanden`

Reply:
not here, since the user is not logged in yet, so it might also be possible, that they are just not exposed, so they exists, but are not available for this request.